src/store/item-ranking.js

```diff
@@ -83,12 +84,48 @@ const actions = {
       commit('setError', "Oh no, it didn't work")
       console.error(err)
     }
+  },
+  moveItemToRankedList({ commit, state }, item) {
+    try {
+      if (state.rankedItems.length >= state.maxRankedItems) {
+        commit(
+          'addUnrankedItem',
+          state.rankedItems[state.rankedItems.length - 1]
+        )
+        commit(
+          'removeRankedItem',
+          state.rankedItems[state.rankedItems.length - 1]
+        )
+      }
+      commit('addRankedItem', item)
+      commit('removeUnrankedItem', item)
+    } catch (err) {
+      commit('setError', err.message)
+    }
+  },
+  moveItemToUnrankedList({ commit }, item) {
+    try {
+      commit('addUnrankedItem', item)
+      commit('removeRankedItem', item)
+    } catch (err) {
+      commit('setError', err.message)
+    }
+  },
+  setRankedItems({ commit, state }, rankedItems = []) {
```

Comment:
We should give this a different name from the mutation. Maybe `setAndLimitRankedItems`?